### PR TITLE
fix to run doctests against NumPy 1.14

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: run_test.py [hunk 1]
```diff
@@ -113,14 +113,16 @@
     elif args.test == 'chainer-doc':
         # See sphinx version RTD uses:
         # https://github.com/rtfd/readthedocs.org/blob/master/requirements/pip.txt
+        # Also note that NumPy 1.14 is required to run doctest, as doctests
```
Comment (hvy) [inline run_test.py]: Strictly speaking `NumPy 1.14 or above` according to the implementation. A minor but do you think we should fix it?

File: run_test.py [hunk 2]
```diff
@@ -191,13 +193,15 @@
     elif args.test == 'cupy-doc':
         # See sphinx version RTD uses:
         # https://github.com/rtfd/readthedocs.org/blob/master/requirements/pip.txt
+        # Also note that NumPy 1.14 is required to run doctest, as doctests
```
Comment (hvy) [inline run_test.py]: Ditto.

**Comment by kmaehashi** [comment]
@hvy Thanks for your review! I revised the comment.

**Comment by hvy** [comment]
Just for the record after offline discussions with @kmaehashi , we've agreed to only test agains latest NumPy (>=1.14) for `chainer-doc`, `cupy-doc`.

**Comment by hvy** [comment]
LGTM!

**Pull request merged**